src/transforms/filter.jl

```diff
@@ -6,6 +6,24 @@
     Filter(function)
 
 Filters the table returning only the rows where the `function` returns true.
```

Comment:
`function` is a Julia keyword, we should rename this to `func` or `pred` for predicate.